addons/hr_timesheet/static/src/scss/timesheets_task_form.scss

```
@@ -1,3 +1,7 @@
 .o_project_kanban .o_kanban_renderer .oe_kanban_align.badge {
     color: inherit;
 }
+
+div[name="unit_amount"] > button {
+    width: 50px !important;
+}
```

We could move that change in `timesheet_uom.scss` in `timesheet_uom` folder and use  the class instead of the name of the field. 

for instance, if it is the `timesheet_uom` widget used then the class will be `o_field_timesheet_uom` and if inside that component we use the float_toggle one then the class in the child element will be `o_field_float_toggle`.

And so we should have something like that:
```suggestion
div.o_field_timesheet_uom > button.o_field_float_toggle {
    width: 50px !important;
}
```

I prefer doing that to be more generic in the case we use the component for another field for instance. 🙂 

However, we will have to do another change in timesheet_uom_timer widget to apply that style, we will have to add `o_field_timesheet_uom` class in `additionalClasses` of that component. If it is not clear, then don't hesitate to ping me, I will help you. 🙂 

I hope everything is clear. 🙂

---

> We could move that change in `timesheet_uom.scss` in `timesheet_uom` folder and use the class instead of the name of the field.
> 
> for instance, if it is the `timesheet_uom` widget used then the class will be `o_field_timesheet_uom` and if inside that component we use the float_toggle one then the class in the child element will be `o_field_float_toggle`.
> 
> And so we should have something like that:
> 
> I prefer doing that to be more generic in the case we use the component for another field for instance. slightly_smiling_face
> 
> However, we will have to do another change in timesheet_uom_timer widget to apply that style, we will have to add `o_field_timesheet_uom` class in `additionalClasses` of that component. If it is not clear, then don't hesitate to ping me, I will help you. slightly_smiling_face
> 
> I hope everything is clear. slightly_smiling_face

I have added class to timesheet_uom_timer widget as per your suggestion [Link](https://github.com/odoo/enterprise/pull/39644) Thanks for review